dex-it/src/test/scala/com/wavesplatform/it/sync/networking/NetworkIssuesTestSuite.scala

```diff
@@ -40,6 +45,33 @@ class NetworkIssuesTestSuite extends WsSuiteBase with HasToxiProxy {
     clearOrderBook()
   }
 
+  "DEXClient should place orders despite of short time disconnect from network" in {
+    val orders = (0 to 100).map { i =>
+      mkOrder(alice, wavesUsdPair, OrderType.SELL, 1.waves, 100 + i)
+    }
+
+    def placeOrders: Future[Unit] =
+      Future
+        .inSeries(orders)(dex1.asyncApi.place(_))
+        .transform {
+          case Success(_) => Try()
+          case Failure(e) => Try(e)
```

Comment:
From the task:
> If we have the bug, **not all orders will be placed** and some requests from Thead_1 will fail by timeout.
That is why it was `Success(Right(()))` and `Success(Left(e))` (okay, there was no `Success`, but it was just an idea)

I thought we need to check this. But it seems `dex1.api.waitForOrderStatus` is necessary. Let's change it to:
```
def placeOrders: Future[Unit] = Future.inSeries(orders)(dex1.asyncApi.place(_).recover { case _ => () })
```

Reply:
ok